lib/get-xo.js

```diff
@@ -0,0 +1,19 @@
+/** @babel */
+import {allowUnsafeNewFunction} from 'loophole';
+import getPackagePath from './get-package-path';
+
+// (base: string) => xo: Promise<Object>
+export default function getXO(base) {
+	const getResolvePath = () => base ? getPackagePath(base) : Promise.resolve();
+
+	return getResolvePath()
```

Comment:
`getXO` needs `getPackagePath`, which actually reads and parses `package.json` files. Before changing all the underlying utils to `sync` counterparts I'd rather do the benchmarks. Do you have an idea on how to go about this?